scripts/upload-screenshots.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const PATH_TO_KEY = process.env.PATH_TO_KEY;
+const BUCKET_NAME = process.env.BUCKET_NAME;
```

Comment:
this is also hard coded. We always upload to the same bucket, so we don't need an environment variable